Doc/library/profile.rst

```diff
@@ -297,6 +297,13 @@ functions:
       Create a :class:`~pstats.Stats` object based on the current
       profile and print the results to stdout.
 
+      The *sort* parameter specifies the sorting order of the displayed
+      statistics. It accepts a single sort key or a tuple of sort keys
+      to enable multi-level sorting.
+
+      .. versionadded:: 3.13
+         :meth:`~Profile.print_stats` can now accept multiple sort arguments.
```

Comment:
"can accept multiple sort arguments" looks confusing. Does `print_stats('calls', 'time')` or `print_stats(sort='calls', sort='time')` work?

Reply:
_Example:_
```python
import cProfile
from pstats import SortKey

with cProfile.Profile() as pr:
    pr.print_stats(sort=("time", "calls"))  # or pr.print_stats(sort=(SortKey.TIME, SortKey.CALLS))
```

I believe that adding a small example to the documentation would make it easier to understand the feature. What do you think?